pages/belt_docs.mdx

```diff
@@ -3,33 +3,114 @@ export default Layout;
 
 # Belt
 
-A stdlib shipped with BuckleScript This stdlib is still in beta but we
-encourage you to try it out and give us feedback.
+The preferred standard library for Reason when targeting JavaScript
```

Comment:
Optimistically, I would see it as a not-yet-complete standard library for targeting JS. ;-)
Maybe we could add something like:

Belt is currently mostly covering collection types, but has no string or date utilities yet. (`Belt.String` is in the works though.)

In the meantime, use

- `Js.String2` for strings
- `Js.Date` for dates

Reply:
Yep, that's probably it!

Reply:
This is completely unsolicited feedback, so feel free to ignore, but I think "preferred" might be too loaded of a word to use with `Belt`.  I think a word more like "default" feels more right to me, since Belt is not a full-on standard library at this point, and it's not globally preferred by everyone. It is however, the library that ships with the compiler, so most people will probably use it at least initially, but I don't think that makes it the preferred stdlib.

Reply:
@andywhite37 Thanks for the feedback, will change it to "default".

Reply:
@andywhite37 Hmm, well, on the other hand, "default" suggests that it is active/used by default - which is not the case, as I need to explicitly open the Belt namespace... 🤔

Reply:
Yeah, maybe default is not quite right. Maybe it could just be `Belt is a standard collections library for Reason`.

Reply:
I already have "recommended" in the next paragraph. Let's just merge it as it is, we can always fine-tune it later.